std/stdio.d

```diff
@@ -536,6 +549,7 @@ Throws: `ErrnoException` in case of error.
                          text("Cannot open file `", name, "' in mode `",
                               stdioOpenmode, "'"));
         }
+        _p = cast(Impl*) enforce(calloc(1, Impl.sizeof), "Out of memory");
```

Comment:
`calloc` is now unnecessary because `initImpl` initializes all fields of `_p` and it doesn't look like anything can fail in-between which would lead to a partial initialization.